src/app/components/charts/workload-profile/workload-profile-panel.component.scss

```
@@ -9,13 +9,214 @@
+  color: white;
+  font-size: 0.875rem;
+  font-weight: 400;
+}
+
+.benchmark-note-icon {
+  @apply h-4 w-auto shrink-0 cursor-help text-[#EAB308];
+}
+
+.workload-profile-breakdown-table {
+  @apply w-full text-white;
+  border-collapse: collapse;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)